### PR TITLE
Change RekeyUI refresh protocol

### PR DESCRIPTION
**Pull request by patrickxb**
Instead of sending KIDs to RekeyUI clients, send Devices.

Updated problemSet in tests to use t_frank, t_george UIDs, KIDs.

r? @maxtaco 
cc: @chrisnojima 

Note that this doesn't close 3100, but @chrisnojima said updated protocol would be useful.

### CODE REVIEW

File: protocol/avdl/rekey.avdl [hunk 1]
```diff
@@ -32,6 +38,12 @@ protocol rekey {
     array<ProblemTLF> tlfs;
   }
 
+  record ProblemSetDevices {
+    User user;
+    KID kid;
+    array<ProblemTLFDevices> tlfs;
```
Comment (maxtaco) [inline protocol/avdl/rekey.avdl]: This is fine but more complicated than i expected.

would it have worked to just expose an map of KID -> Devices in addition to the prexisting ProblemSet?

Comment (maxtaco) [inline protocol/avdl/rekey.avdl]: In other words, this is what I had in mind:

```record ProblemSetDevices {
   ProblemSet problemSet;
   array<Device> devices;
}```

Then electron can perform the mapping, and we don't list the same device over and over again...

Comment (patrickxb) [inline protocol/avdl/rekey.avdl]: I was trying to make it so electron could just display the data it needs to display like the interface mockup in zeplin and not need to worry about what a KID is and how it relates to a Device.

@chrisnojima do you have any preference?

Comment (chrisnojima) [inline protocol/avdl/rekey.avdl]: doesn't matter really, simple for us to do the mapping if its kid-> device

**Comment by patrickxb** [comment]
@maxtaco @chrisnojima PTAL

**Comment by maxtaco** [comment]
All good! Thanks!

**Pull request merged**